client-data.md

```diff
@@ -0,0 +1,34 @@
+# Filecoin Client Data Processing
+
+Data that clients wish to store must be pre-processed before it can be sent to miners and have a storage deal made for it. 
+
+This processing is comprised of two top-level pieces:
+
+1. **Transfer Encoding**
+2. **Storage Encoding**
+
+Transfer Encoding is a pre-processing for the file to allow it to be transferred over the network more easily. Both for the initial transfer to the storage miner, and for later retrievals. This mechanism can be done in any way that is supported by the client and the miner (alternative strategies may be implemented).
+
+Storage Encoding is a pre-processing for the file that allows the Filecoin storage market and proofs to reference the data. Filecoin operates on a special type of merkle tree generated with a function we call `RepHash`. The root hash of this merkle tree is called the `Piece Commitment` or `CommP` and is included in the deal when it is proposed to a miner.
+
+The next sections will go into some additional detail on how each of these encodings works.
+
+## Transfer Encoding
+
+Transfer Encoding may be computed in a variety of ways, as long as it is supported by both the client and miner involved. Here, we document the `default` transfer encoding.
```

Comment:
Dumb question: Does our reference implementation use Unixfsv1 today? I have not yet delved into the guts of the blockservice / bitswap combo (the current client-to-miner transfer mechanism).

Reply:
yeah, thats what we currently use.